### PR TITLE
feat(rollkit): add rollkit init

### PR DESCRIPTION
**Pull request by julienrbrt**
The rollkit app can successfully replace these steps:
This PR is for as well abstracting the configuration of the sequencer in the app.

Usage `ignite rollkit init` (as replacement of `ignite chain init` that is used for normal cosmos-sdk chains)

demo: https://asciinema.org/a/661669 using [local-da](https://github.com/rollkit/local-da)

### CODE REVIEW

**Comment by julienrbrt** [comment]
https://asciinema.org/a/661669

File: rollkit/cmd/add.go [hunk 1]
```diff
@@ -40,7 +41,7 @@ func NewRollkitAdd() *cobra.Command {
 				return err
 			}
 
-			chain, err := chain.New(absPath)
+			chain, err := chain.New(absPath, chain.CollectEvents(session.EventBus()))
```
Comment (Pantani) [inline rollkit/cmd/add.go:44]:
```suggestion
			c, err := chain.New(absPath, chain.CollectEvents(session.EventBus()))
```

File: rollkit/cmd/add.go [hunk 2]
```diff
@@ -59,7 +60,7 @@ func NewRollkitAdd() *cobra.Command {
 				return err
 			}
 
-			session.Printf("\n🎉 RollKit added (`%[1]v`).\n\n", chain.AppPath())
+			session.Printf("🎉 RollKit added (`%[1]v`).\n", chain.AppPath(), chain.Name())
 			return nil
```
Comment (Pantani) [inline rollkit/cmd/add.go:64]:
```suggestion
			return session.Printf("🎉 RollKit added (`%[1]v`) for chain %[2]v.\n", chain.AppPath(), chain.Name())
```

File: rollkit/cmd/init.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+			if err != nil {
+				return err
+			}
+
+			rc, err := chain.New(absPath, chain.CollectEvents(session.EventBus()))
+			if err != nil {
+				return err
+			}
+
+			if localDa, err := cmd.Flags().GetBool(flagLocalDa); err == nil && localDa {
```
Comment (Pantani) [inline rollkit/cmd/init.go:44]:
```suggestion
			if localDa, _ := cmd.Flags().GetBool(flagLocalDa); localDa {
```

File: rollkit/cmd/init.go [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+				igniteConfig, err := rc.Config()
+				if err != nil {
+					return err
+				}
+
+				igniteConfig.Validators[0].Bonded = "1000000000stake"
```
Comment (Pantani) [inline rollkit/cmd/init.go:50]: May we use a flag for the bonded and balance and use these values as minimum

File: rollkit/cmd/init.go [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+// getPubKey returns the validator public key.
+func getPubKey(chainHome string) (crypto.PubKey, error) {
+	keyFilePath := filepath.Join(chainHome, "config", "priv_validator_key.json")
+	keyJSONBytes, err := os.ReadFile(keyFilePath)
+	if err != nil {
+		return nil, err
+	}
+	var pvKey cmprivval.FilePVKey
+	err = cmtjson.Unmarshal(keyJSONBytes, &pvKey)
+	if err != nil {
```
Comment (Pantani) [inline rollkit/cmd/init.go:119]:
```suggestion
	if err = cmtjson.Unmarshal(keyJSONBytes, &pvKey); err != nil {
```

File: rollkit/cmd/init.go [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+	c.Flags().StringP(flagPath, "p", ".", "path of the app")
+	c.Flags().BoolP(flagLocalDa, "l", false, "use local-da (creates the expected genesis account for local-da)")
+
+	return c
+}
+
+// getPubKey returns the validator public key.
+func getPubKey(chainHome string) (crypto.PubKey, error) {
+	keyFilePath := filepath.Join(chainHome, "config", "priv_validator_key.json")
```
Comment (Pantani) [inline rollkit/cmd/init.go:112]: Is it possible to use a flag from another path? Should we also create a flag for it?

Comment (julienrbrt) [reply rollkit/cmd/init.go:112]: Another flag isn't useful here, as the they are always created there and expected there.

File: rollkit/cmd/init.go (hunk 3 above)
Comment (julienrbrt) [reply rollkit/cmd/init.go:44]: Why ignoring the error?

File: rollkit/cmd/init.go (hunk 4 above)
Comment (julienrbrt) [reply rollkit/cmd/init.go:50]: This value is expected by local-da, changing it will break it.

**Review by Pantani — APPROVED** [review]

**Pull request merged**